utils/appliance.py

```diff
@@ -1628,6 +1628,56 @@ def reset_automate_model(self):
         with self.ssh_client as ssh_client:
             ssh_client.run_rake_command("evm:automate:reset")
 
+    def configure_appliance_for_openldap_ext_auth(self, appliance_name):
+        """Sets up the appliance for an external authentication with OpenLdap.
+
+        Keywords:
+            get_groups: Get User Groups from External Authentication (httpd).
+            ipaserver: IPA server address.
+            iparealm: Realm.
+            credentials: Key of the credential in credentials.yaml
+        """
+        openldap_domain1 = conf.cfme_data['openldap_domain1']
+        appliance_fqdn = '{}.{}'.format(appliance_name, openldap_domain1['domain_name'])
+        # updating the /etc/hosts is a workaround due to the
+        # https://bugzilla.redhat.com/show_bug.cgi?id=1360928
+        ssh = self.ssh_client
```

Comment:
@psav :) yes, that's not good. changed it in new patch. changes will be pushed to this PR.